syntax = "proto3";

import "wrappers.proto";

message PresetParameters {
  google.protobuf.Int32Value amp_gain = 1;
  google.protobuf.Int32Value amp_channel = 2;
  google.protobuf.Int32Value bass = 3;
  google.protobuf.Int32Value middle = 4;
  google.protobuf.Int32Value treble = 5;
  google.protobuf.Int32Value contour = 6;
  google.protobuf.Int32Value volume = 7;
  google.protobuf.Int32Value noise_gate_threshold = 8;
  google.protobuf.Int32Value noise_gate_hysteresis = 9;
  google.protobuf.Int32Value noise_gate_attack = 10;
  google.protobuf.Int32Value noise_gate_hold = 11;
  google.protobuf.Int32Value noise_gate_release = 12;
  google.protobuf.Int32Value noise_gate_bypass = 13;
  google.protobuf.Int32Value chorus_rate = 14;
  google.protobuf.Int32Value chorus_depth = 15;
  google.protobuf.Int32Value chorus_mix = 16;
  google.protobuf.Int32Value chorus_bypass = 17;
  google.protobuf.Int32Value wah_position = 18;
  google.protobuf.Int32Value wah_vocal = 19;
  google.protobuf.Int32Value wah_bypass = 20;
}